lib/aws/ses-calls.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Brigham Young University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+const AWS = require('aws-sdk');
+const winston = require('winston');
+
+function verifyEmailAddress(address) {
```

Comment:
I don't see a test on this function, could we get one added before I merge the PR?

Reply:
Okay, I added those tests.